client/src/main/scala/org/apache/celeborn/client/commit/ReducePartitionCommitHandler.scala

```diff
@@ -391,8 +405,9 @@ class ReducePartitionCommitHandler(
     shuffleMapperAttempts.synchronized {
       if (!shuffleMapperAttempts.containsKey(shuffleId)) {
         val attempts = new Array[Int](numMappers)
-        0 until numMappers foreach (idx => attempts(idx) = -1)
+        util.Arrays.fill(attempts, -1)
```

Comment:
`Arrays.fill` is faster than the `foreach` - while not the focus of this PR, included it when I observed it.